include/forcing/Forcing.h

```diff
@@ -102,7 +102,7 @@ class Forcing
      */
     double get_current_hourly_precipitation_meters_per_second()
     {
-        return precipitation_rate_meters_per_second_vector[forcing_vector_index_ptr];
+        return precipitation_rate_meters_per_second_vector.at(forcing_vector_index_ptr);
```

Comment:
The .at() is less efficient. Do we want this on top of having the bounds check in the get_next function?

Reply:
I don't think it is important, but you would have to check the entire range and handle both side, `< 0` and `> .size()`

Reply:
Okay thanks. I will leave the .at() in there.